test/integration/serverTest.js

```diff
@@ -154,6 +154,126 @@ describe('Brave Sensor server', () => {
     await redis.disconnect()
   })
 
+  describe('POST request: alerts from firmware state machine for locations with a non-null Particle Siren Id', () => {
+    beforeEach(async () => {
+      await redis.clearKeys()
+      await db.clearSessions()
+      await db.clearLocations()
+      await db.clearClients()
+      const client = await clientFactory(db)
+      await db.createLocation(
+        testLocation1Id,
+        testLocation1PhoneNumber,
+        MOVEMENT_THRESHOLD,
+        STILLNESS_TIMER,
+        DURATION_TIMER,
+        1000,
+        INITIAL_TIMER,
+        ['+15005550006'],
+        '+15005550006',
+        ['+15005550006'],
+        1000,
+        'locationName',
+        door_coreID,
+        radar_coreID,
+        'XeThru',
+        'alertApiKey',
+        true,
+        true,
+        'particleCoreIdTest',
+        client.id,
+      )
+      sandbox.stub(braveAlerter, 'startAlertSession')
+      sandbox.stub(braveAlerter, 'sendSingleAlert')
+      sandbox.spy(particle, 'callFunction')
+    })
+    afterEach(async () => {
+      await redis.clearKeys()
+      await db.clearSessions()
+      await db.clearLocations()
+      await db.clearClients() 
+      sandbox.restore()
+      helpers.log('\n')
+    })
+    
+    it('should return 200 to a request with no headers', async () => {
+      const response = await chai.request(server).post('/api/sensorEvent').send({})
+      expect(response).to.have.status(200)
+    })
+
+    it('should call particle.callFunction with the correct device ID if sirenParticleId is not null, and return 200', async () => {
+      const response = await firmwareAlert(radar_coreID, SENSOR_EVENT.STILLNESS)
+      expect(response).to.have.status(200)
+      expect(particle.callFunction).to.have.been.calledWith('particleCoreIdTest', helpers.getEnvVar('PARTICLE_ACCESS_TOKEN_TEST'))
```

Comment:
I think actually here you want to use just helpers.getEnvVar('PARTICLE_ACCESS_TOKEN_TEST'), because helpers.getEnvVar adds '_TEST' when NODE_ENV=test, as it does when we run npm test

Reply:
Do you mean replace line 207 with just helpers.getEnvVar('PARTICLE_ACCESS_TOkEN_TEST'),  or replace just the body of the calledWith() function?

Reply:
I meant replace the second parameter in the calledWith() function with just helpers.getEnvVar('PARTICLE_ACCESS_TOKEN)